client/src/cb-admin/pages/Login.js

```diff
@@ -40,19 +39,20 @@ export default class Login extends React.Component {
 
     CbAdmin.login({ email: this.state.email, password: this.state.password })
       .then((res) => {
-        localStorage.setItem('token', res.data.result.token);
+        if (process.env.NODE_ENV !== 'test') localStorage.setItem('token', res.data.result.token);
```

Comment:
Is this stripped out by babel at build time? `process` is a Node API and will cause an exception on the browser.

Reply:
This was working fine locally, (ie no errors and you can actually console.log(process.env) in the browser), but I guess this is something to do with CRA's hot reloading. Any suggestions of a better approach? if not I could probs do:
```
if(process && process...)
```